formatters/html/html_test.go

```diff
@@ -108,6 +108,39 @@ func TestTableLineNumberNewlines(t *testing.T) {
 </span>`)
 }
 
+func TestLinkeableLineNumbers(t *testing.T) {
+	f := New(WithClasses(true), WithLineNumbers(true), LinkableLineNumbers(true, "line"))
+	it, err := lexers.Get("go").Tokenise(nil, "package main\nfunc main()\n{\nprintln(\"hello world\")\n}\n")
+	assert.NoError(t, err)
+
+	var buf bytes.Buffer
+	err = f.Format(&buf, styles.Fallback, it)
+	assert.NoError(t, err)
+
+	assert.Equal(t, buf.String(), `<pre class="chroma"><span class="ln" id="line1"><a style="outline: none; text-decoration:none; color:inherit" href="#line1">1</a></span><span class="kn">package</span> <span class="nx">main</span>
```

Comment:
I think this could be a bit annoying to maintain, can you just do a substring check for the links? Here and below?

Reply:
Updated, but I kept the `id` attribute in the substring that is searched for as there are no other tests that verify this, WDYT?